package.json

```diff
@@ -58,7 +58,7 @@
     "flow-bin": "^0.129.0",
     "husky": "^4.2.5",
     "istanbul": "^0.4.5",
-    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#master",
```

Comment:
Why not master branch?

Reply:
because i want to point to a specific version

Reply:
but this is just for dev environment , you should point to the master